File: LeetCode/Easy/1-two-sum/try-2.py
/*
 * Problem #1: Two Sum
 * Difficulty: Easy
 * Submission: Try 2
 * status: Accepted
 * Language: python
 * Date: 7/4/2026, 6:49:02 PM
 * Link: https://leetcode.com/problems/two-sum/
 */

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        seen = {}  # value -> index
        for i, num in enumerate(nums):
            complement = target - num
            if complement in seen:
                return [seen[complement], i]
            seen[num] = i

File: LeetCode/Easy/58-length-of-last-word/try-2.java
/*
 * Problem #58: Length of Last Word
 * Difficulty: Easy
 * Submission: Try 2
 * status: Accepted
 * Language: java
 * Date: 7/15/2026, 1:56:59 PM
 * Link: https://leetcode.com/problems/length-of-last-word/
 */

class Solution {
    public int lengthOfLastWord(String s) {
        String[] arr = s.trim().split(" ");

        return arr[arr.length - 1].length();
    }
}

   
